test/wasm-spec/core/bulk.wast

```diff
@@ -219,7 +219,7 @@
 (assert_trap (invoke "init" (i32.const 2) (i32.const 0) (i32.const 2))
     "out of bounds table access")
 (assert_trap (invoke "call" (i32.const 2))
-    "uninitialized element 2")
+    "uninitialized element")
```

Comment:
IMO it's just a part of message presenting that table element at index `2` is not yet initialzed.

Reply:
Comparing to the origin version, there is still the same message comment
https://github.com/WebAssembly/spec/blob/905f42d3e4d03a44614bee85d50cab988d55869d/test/core/bulk.wast#L222

Reply:
Shall we report this to them as well?

Reply:
This is the only case where they used this 2, no other assert trap has such information.

Reply:
I think that it's not an error nor typo, just a message presenting the reason of error.
IMO It seems that we can ignore this because there is no problem (it's fine as it is)